internal/agentdeployer/info.go

```diff
@@ -71,6 +71,17 @@ type AgentInfo struct {
 			// Name prefix for the host's name
 			NamePrefix string
 		}
+
+		// User user to run Elastic Agent process
+		User string
+		// PidMode selects the host PID mode
+		// (From docker-compose docs) Turns on sharing between container and the host
+		// operating system the PID address space
+		PidMode string
+		// Runtime is the selected runtime to run the Elastic Agent process
+		Runtime string
+		// Capabilities is a list of the capabilities needed to run the Elastic Agent process
+		Capabilities []string
```

Comment:
Nit. Maybe we can call this `LinuxCapabilities` to avoid confusion with other capabilities.
```suggestion
		// LinuxCapabilities is a list of the capabilities needed to run the Elastic Agent process
		LinuxCapabilities []string
```

Reply:
Agreed, do you think it should be updated that naming in the configuration file for system tests ? @jsoriano 

Currently, those settings in the configuration file looks like:
```yaml
agent:
  runtime: docker
  user: "root"
  pid_mode: "host"
  capabilities:
    - AUDIT_CONTROL
    - AUDIT_READ
```
```diff
   runtime: docker
   user: "root"
   pid_mode: "host"
-  capabilities:
+  linux_capabilities:
     - AUDIT_CONTROL
     - AUDIT_READ
```

Reply:
Yes, call them `linux_capabilities` in the config.